pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlUtils.java

```diff
@@ -67,9 +67,12 @@ public static void validatePermission(@Nullable String tableName, AccessType acc
     } catch (WebApplicationException exception) {
       // throwing the exception if it's WebApplicationException
       throw exception;
-    } catch (Exception e) {
+    } catch (Throwable t) {
+      // catch and log Throwable for NoSuchMethodError which can happen when there are classpath conflicts
+      // otherwise, grizzly will return a 500 without any logs or indication of what failed
+      LOGGER.error("Caught exception while validating permission for {}", userMessage, t);
```

Comment:
We don't need to log error here. It will be logged within the `ControllerApplicationException` constructor